apiserver/cmd/main.go

```diff
@@ -58,6 +58,7 @@ func startRpcServer(resourceManager *manager.ResourceManager) {
 		grpc.MaxRecvMsgSize(math.MaxInt32))
 	api.RegisterClusterServiceServer(s, server.NewClusterServer(resourceManager, &server.ClusterServerOptions{CollectMetrics: *collectMetricsFlag}))
 	api.RegisterComputeTemplateServiceServer(s, server.NewComputeTemplateServer(resourceManager, &server.ComputeTemplateServerOptions{CollectMetrics: *collectMetricsFlag}))
+	api.RegisterRayJobServiceServer(s, server.NewRayJobServer(resourceManager))
```

Comment:
Even collect metrics is not being used yet, let's keep the code structure same?

Reply:
fixed